awscli/paramfile.py

```diff
@@ -107,6 +107,8 @@
     'service-catalog.create-product.support-url',
     'service-catalog.update-product.support-url',
 
+    'sesv2.put-account-details.website-url',
```

Comment:
So one not so great gotcha (that we should change) is that for the prefix of these event names, we use the endpoint prefix. For `sesv2` the endpoint prefix is `ses`, so the event should be `ses.put-account-details.website-url`.

I think for this one because of this gotcha, we should add a test for it. It would be great if we can add a test like this one: https://github.com/aws/aws-cli/blob/develop/tests/functional/iam/test_create_open_id_connect_provider.py where we make sure the value used is the URL instead of its contents.